custom_components/tuya_local/devices/DH-CSK03W_dehumidifier.yaml

```diff
@@ -0,0 +1,95 @@
+name: "DH-CSK03W Dehumidifier"
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
```

Comment:
Custom icons should not be needed. Some older dehumidifier configs provide them because the tank indicator is combined with the humidifier, so to get the icon priorities right they are all specified even when they match the default. but since binary_sensors have been introduced the tank indicator is generally separate.

Reply:
Removed mappings